package.json

```diff
@@ -67,8 +78,8 @@
     "test:php": "./vendor/bin/phpunit -c phpunit.xml"
   },
   "engines": {
-    "node": ">=8.9.3",
-    "npm": ">=5.5.1"
+    "node": ">=12.0.0",
+    "npm": ">=6.0.0"
```

Comment:
@wordpress/scripts has a hard node version requirement of >12 so I've bumped the engine requirements here.